### PR TITLE
Add tests for ReplicationRegion

### PR DESCRIPTION
**Pull request by xirc**

### CODE REVIEW

File: core/src/multi-jvm/scala/lerna/akka/entityreplication/ReplicationRegionInitializingSpec.scala [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+        }
+      }
+
+      "stash messages in the initializing-1." in {
+        runOn(node1) {
+          replicationRegion ! ReplicationRegion.Broadcast(
+            RaftProtocol.Command(
+              TestReplicationActor.Message(myself, "initializing-1"),
+            ),
+          )
```
Comment (negokaz) [inline core/src/multi-jvm/scala/lerna/akka/entityreplication/ReplicationRegionInitializingSpec.scala:171]: @xirc 
How about verifying that `raftActorProbe` doesn't receive any messages explicitly?
We could be sure that ReplicationRegion stashes it.

**Review by negokaz — COMMENTED** [review]
Great👍 I left a comment.

File: core/src/multi-jvm/scala/lerna/akka/entityreplication/ReplicationRegionInitializingSpec.scala (hunk 1 above)
Comment (xirc) [reply core/src/multi-jvm/scala/lerna/akka/entityreplication/ReplicationRegionInitializingSpec.scala:171]: @negokaz 
Thank you!
I've changed the test: 49e5833

**Review by negokaz — APPROVED** [review]
LGTM 👍

**Pull request merged**